lxqt-config-appearance/lxqtthemeconfig.ui

```diff
@@ -51,6 +51,13 @@
      </column>
     </widget>
    </item>
+   <item>
+    <widget class="QCheckBox" name="wallpaperOverride">
+     <property name="text">
+      <string>Override independently set wallper</string>
```

Comment:
"Use theme's wallpaper when changing themes.", or something like that. "Forcing" would seem like the user could't change it.